README.md

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 
 🤗 Optimum Habana is the interface between the 🤗 Transformers library and [Habana's Gaudi processor (HPU)](https://docs.habana.ai/en/latest/index.html).
 It provides a set of tools enabling easy model loading and fine-tuning on single- and multi-HPU settings for different downstream tasks.
-The current release focuses on question answering and text classification and enables users to try other models for other tasks with only a few changes.
+The list of officially supported models and tasks is available [here](https://github.com/huggingface/optimum-habana#validated-models). Users can try other models and tasks with only few changes.
```

Comment:
instead of supported, should we say sth different like "verified"?

Reply:
I remember in the README we put "validated", shall I put this?

Reply:
yes